<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vehiculeUI</class>
 <widget class="QDialog" name="vehiculeUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1039</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>60</y>
     <width>811</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>171</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="nums">
      <property name="placeholderText">
       <string>nums</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="numc">
      <property name="placeholderText">
       <string>numc</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateEdit" name="dateEdit"/>
    </item>
    <item>
     <widget class="QLineEdit" name="km">
      <property name="placeholderText">
       <string>km</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="tel">
      <property name="placeholderText">
       <string>tel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="modele">
      <property name="placeholderText">
       <string>modele</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="marque">
      <property name="placeholderText">
       <string>marque</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="nom">
      <property name="placeholderText">
       <string>nom</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>171</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="ajouter">
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="editer">
      <property name="text">
       <string>Editer</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>360</y>
     <width>241</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLineEdit" name="lineEdit_supp">
      <property name="placeholderText">
       <string>Supprimer</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="supp">
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
